src/research-tracks.ts

```diff
@@ -12,7 +12,7 @@ export default {
     [], ["q"], ["q"], ["2q,3pw"], ["2q"], ["4q"]
   ],
   [ResearchField.GaiaProject]: [
-    [], ["gf"], ["3t"], ["3pw", "gf"], ["gf"], ["4vp", "g > vp"]
+    [], [">gf", "gf >> 6tg"], ["3t", "gf >> 6tg"], ["3pw", ">gf", "gf >> 4tg"], [">gf", "gf >> 3tg"], ["4vp", "gf >> 3tg", "g > vp"]
```

Comment:
I don't understand the `gf >> 6tg`. It's not used in the code, and research areas already have tooltips.

Unless you mean to gave the gaiaformer's cost based on the event?